py_modules/caller.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import os
+
+# pipeline modules are using relative importing
+# so using sys.path hack here
+from ecephys_spike_sorting import scripts
+sys.path.append(os.path.dirname(scripts.__file__))
```

Comment:
Manipulating the Python `sys.path` inside a module is kind of a hack. Could this be done in some higher-level library initialization stuff instead?